src/main/java/com/yahoo/bullet/result/Clip.java

```diff
@@ -22,8 +23,8 @@
     private Meta meta = new Meta();
     private List<BulletRecord> records = new ArrayList<>();
 
-    private static Map<String, Object> asMap(BulletRecord record) {
-        Map<String, Object> mapped = new HashMap<>();
+    private static <T extends Serializable> Map<String, T> asMap(BulletRecord<T> record) {
```

Comment:
Probably should make this public if we want to get rid of JSONFormatter. Not sure, what do you think? Should we leave GSON in bullet-core?